ceph_iscsi_config/target.py

```diff
@@ -53,6 +53,7 @@ def __init__(self, logger, iqn, gateway_ip_list, enable_portal=True):
 
         self.enable_portal = enable_portal  # boolean to trigger portal IP creation
         self.logger = logger                # logger object
+        self.config = config
```

Comment:
The classes that are based on the GWObject have a Config object already.

Reply:
Good spotting. This is fixed now, thanks.